CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
```

Comment:
There is a NBSP between `###` and `Added`. I don't know why, but you keep inserting one quite consistently.